ci-operator/step-registry/ipi/conf/azure/provisioned-des/ipi-conf-azure-provisioned-des-commands.sh

```diff
@@ -27,22 +27,13 @@ PATCH="/tmp/install-config-provisioned-des.yaml.patch"
 
 # create a patch with existing des
 cat > "${PATCH}" << EOF
-compute:
-- platform:
-    azure:
+platform:
+  azure:
+    defaultMachinePlatform:
```

Comment:
This change can be applied to older version of OCP?

Reply:
Jinyun, I think you can consider introducing a new var to control whether use `compute/controlPlane` or `defaultMachinePlatform`, the default would be the _existing behavior_ .